services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -376,3 +376,24 @@ export const useReleasesForApp = (app: string) =>
             a.version === -1 ? -1 : b.version === -1 ? 1 : b.version - a.version
         )
     );
+
+// Get all releases for an app and group them by date
+export const useReleasesForAppGroupByDate = (app: string) =>
```

Comment:
please add a test for this function

Reply:
This function doesn't work here, so I replaced it with a function in the component's code